### PR TITLE
fix phing target descriptions

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| The descriptions were just wrong and misleading.
|New feature| No 
|BC breaks| No 
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: project-base/build.xml [hunk 1]
```diff
@@ -54,9 +54,9 @@
...
-    <target name="build-new" depends="wipe,composer,npm,dirs-create,domains-urls-check,assets,db-rebuild,grunt,error-pages-generate,warmup" description="Builds application for production preserving your DB."/>
-    <target name="build-demo" depends="wipe,composer,npm,dirs-create,domains-urls-check,assets,db-demo,grunt,error-pages-generate,warmup" description="Builds application for production with clean DB."/>
-    <target name="db-demo" depends="db-wipe-public-schema,db-import-basic-structure,db-migrations,db-fixtures-demo-singledomain,create-domains-data,db-fixtures-demo-multidomain,load-plugin-demo-data,generate-friendly-urls,replace-domains-urls" description="Builds application for production with clean demo DB."/>
+    <target name="build-new" depends="wipe,composer,npm,dirs-create,domains-urls-check,assets,db-rebuild,grunt,error-pages-generate,warmup" description="Builds application for production with clean DB (with base data only)."/>
+    <target name="build-demo" depends="wipe,composer,npm,dirs-create,domains-urls-check,assets,db-demo,grunt,error-pages-generate,warmup" description="Builds application for production and fills DB with demo data."/>
```
Comment (PetrHeinz) [inline project-base/build.xml]: Sound like the DB is preserved and demo data are added. I would suggest "Builds application for production **with clean demo DB**"

Comment (vitek-rostislav) [reply project-base/build.xml]: Are you sure? The target depends on `db-demo` which depends on `db-wipe-public-schema` that actually drops the public schema so I believe no data are preserved. Or am I missing something?

Comment (PetrHeinz) [reply project-base/build.xml]: I mean the new description sounds like the DB is preserved and filled with demo data:

> Builds application for production and fills DB with demo data.

That's why I suggested using this message instead:

> Builds application for production with clean demo DB

Comment (vitek-rostislav) [reply project-base/build.xml]: Oh, I see :laughing: thx, fixed :+1:

**Pull request merged**